### ISSUE TITLE
architecture_601 fix not applied in with/select statement

### ISSUE THREAD

**Issue by maltaisn** [jeremiah-c-leary/vhdl-style-guide#1448]
This file:
```vhdl
library ieee;
  use ieee.std_logic_1164.all;

entity b is
  port (
    aBc : in    std_logic;
    DeF : out   std_logic
  );
end entity b;

architecture a of b is

begin

  def <= ABC;   -- Fixed

  with ABC select def <=   -- Not fixed
    '0' when '1',
    '1' when '0',
    'X' when others;

end architecture a;
```
With the following rules:
```yml
rule:
  port_010:
    disable: yes
```
The case mismatch is reported and fixed in the assignment:
```
Phase 6 of 7... Reporting
Total Rules Checked: 877
Total Violations:    2
  Error   :     2
  Warning :     0
-------------------+------------+------------+--------------------------------------
  Rule             |  severity  |  line(s)   | Solution
-------------------+------------+------------+--------------------------------------
  architecture_601 | Error      |         15 | Port case mismatch:  Change def to DeF
  architecture_601 | Error      |         15 | Port case mismatch:  Change ABC to aBc
-------------------+------------+------------+--------------------------------------
```
but not in the with/select statement.

**Comment by maltaisn** [comment]
The case is also not fixed when used in a subprogram:

```vhd
library ieee;
  use ieee.std_logic_1164.all;

entity b is
  port (
    a : in    std_logic;
    b : out   std_logic
  );
end entity b;

architecture a of b is

  function bar return std_logic is
  begin

    return A;  -- Not fixed

  end function bar;

begin

  b <= a;  -- Fixed

end architecture a;
```
(with default rules this time)
